src/main/test/CocktailControllerTest.java

```diff
@@ -0,0 +1,138 @@
+
+    @MockBean
+    private CocktailService cocktailService;
+
+
+    private Photo photoList = new Photo(1L, "http//google.com", "", "");
+    private List<Ingredient> ingredientList = Arrays.asList(new Ingredient(1L, "Wodka", "4 cl Wodka"), new Ingredient(2L, "Pfirsich Likör", "4 cl Pfirsich Likör"), new Ingredient(3L, "cranberry juice or Cranberry Rectal", "8 cl cranberry juice or Cranberry Rectal"), new Ingredient(4L, "orange juice", "8 cl of orange juice"), new Ingredient(5L, "ice", "ice cubes"));
+    private Cocktail cocktail = new Cocktail(1L, "Sex on the Beach Cocktail Recipe", "So I have to say one of the best drink's I've drunk so far has become my favorite drink as well", null, 1, 1, new User(1L, "Iurie Railean"), photoList, ingredientList);
+
+    private CocktailWebOutput cocktailWebOutput = new CocktailWebOutput(1l, COCKTAIL_NAME, "", null, new UserWeb(COCKTAIL_AUTHOR_NAME), Arrays.asList(new IngredientWeb(INGRDIENT_NAME, "some description")), 1, 1, null);
+    private List<CocktailWebOutput> cocktailWebOutputList = Arrays.asList(cocktailWebOutput);
+
+    private List<Cocktail> cocktailList = Arrays.asList(cocktail);
+
+    private final static String COCKTAIL_NAME = "Sex on the Beach Cocktail Recipe";
+    private final static String COCKTAIL_AUTHOR_NAME = "Iurie Railean";
+    private final static String INGRDIENT_NAME = "Wodka";
+
+    private final static String COCKTAIL_WEB_ARRAY_JSON = "[{\"id\":1,\"title\":\"Sex on the Beach Cocktail Recipe\",\"announce\":\"So I have to say one of the best drink's I've drunk so far has become my favorite drink as well\",\"publicationDate\":null,\"prepTime\":1,\"cookingTime\":1,\"author\":{\"id\":1,\"name\":\"Iurie Railean\"},\"image\":{\"id\":1,\"path\":\"http//google.com\",\"title\":\"\",\"alt\":\"\"},\"ingredients\":[{\"id\":1,\"title\":\"Wodka\",\"description\":\"4 cl Wodka\"},{\"id\":2,\"title\":\"Pfirsich Likör\",\"description\":\"4 cl Pfirsich Likör\"},{\"id\":3,\"title\":\"cranberry juice or Cranberry Rectal\",\"description\":\"8 cl cranberry juice or Cranberry Rectal\"},{\"id\":4,\"title\":\"orange juice\",\"description\":\"8 cl of orange juice\"},{\"id\":5,\"title\":\"ice\",\"description\":\"ice cubes\"}]}]";
+
+    private final static String COCKTAIL_SEX_ARRAY_JSON = "[{\"id\":1,\"title\":\"Sex on the Beach Cocktail Recipe\",\"announce\":\"So I have to say one of the best drink's I've drunk so far has become my favorite drink as well\",\"publicationDate\":null,\"author\":{\"name\":\"Iurie Railean\"},\"ingredients\":[{\"title\":\"Wodka\",\"description\":\"4 cl Wodka\"},{\"title\":\"Pfirsich Likör\",\"description\":\"4 cl Pfirsich Likör\"},{\"title\":\"cranberry juice or Cranberry Rectal\",\"description\":\"8 cl cranberry juice or Cranberry Rectal\"},{\"title\":\"orange juice\",\"description\":\"8 cl of orange juice\"},{\"title\":\"ice\",\"description\":\"ice cubes\"}],\"prepTimeMinute\":0,\"cookingTime\":1,\"image\":{\"path\":\"http//google.com\",\"title\":\"\",\"alt\":\"\"}}]";
```

Comment:
I adjusted these strings to test with mocked cocktailRepository, but you should also drop this and next lines and replace it with json strings of you CocktailWebOutput objects, since this is what CocktailService returns.